### PR TITLE
Profile dropdown button created

### PR DESCRIPTION
**Pull request by kushwahramkumar2003**
# Display profile button After Login

## Approach

I have used the Shadcn dropdown menu to fix the problem where the login button remains visible after logging. The dropdown menu now displays user information, including the user's profile image, email, name, and a logout button.

## Video

https://github.com/user-attachments/assets/5ab6b1d8-0091-4913-b469-ceded4b3fa95

### CODE REVIEW

**Comment by kushwahramkumar2003** [comment]
@VineeTagarwaL-code please review my pr

File: components/UserImage.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+export default function UserImage({ image }: any) {
+    return (
+        <div>
+            <img
```
Comment (VineeTagarwaL-code) [inline components/UserImage.tsx:6]: USE NEXT IMAGE TAG

**Review by VineeTagarwaL-code — CHANGES_REQUESTED** [review]
make the changes asked, thank you

File: components/UserImage.tsx (hunk 1 above)
Comment (kushwahramkumar2003) [reply components/UserImage.tsx:6]: Ok i Will do

**Review by VineeTagarwaL-code — APPROVED** [review]

**Pull request merged**